src/common/fam_libfabric.cpp

```diff
@@ -275,14 +275,29 @@ int fabric_initialize(const char *name, const char *service, bool source,
             (struct fi_ep_attr *)calloc(1, sizeof(struct fi_ep_attr));
     hints->ep_attr->type = FI_EP_RDM;
 
-    hints->caps = FI_MSG | FI_RMA | FI_RMA_EVENT | FI_ATOMICS;
+    if (!hints->tx_attr)
+        hints->tx_attr =
+            (struct fi_tx_attr *)calloc(1, sizeof(struct fi_tx_attr));
+    hints->tx_attr->op_flags = FI_DELIVERY_COMPLETE;
+
+    hints->caps = FI_MSG | FI_RMA | FI_ATOMICS;
```

Comment:
is it safer to add FI_REMOTE_WRITE | FI_REMOTE_READ? and also can we have a macro for this as its used in 2 places?